downstream/attributes/attributes.adoc

```diff
@@ -277,94 +277,154 @@
 //
 // titles/aap-operator-backup
 :TitleOperatorBackup: Backup and recovery for operator environments
+:URLOperatorBackup: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/backup_and_recovery_for_operator_environments
+:LinkOperatorBackup: {URLOperatorBackup}[{TitleOperatorBackup}]
 //
 // titles/central-auth
 :TitleCentralAuth: Access management and authentication
+:URLCentralAuth: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/access_management_and_authentication
+:LinkCentralAuth: {URLCentralAuth}[{TitleCentralAuth}]
 //
 // titles/getting-started
 :TitleGettingStarted: Getting started with Ansible Automation Platform
+:URLGettingStarted: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/getting_started_with_ansible_automation_platform
+:LinkGettingStarted: {URLGettingStarted}[{TitleGettingStarted}]
 //
 // titles/aap-containerized-install
 :TitleContainerizedInstall: Containerized installation
+:URLContainerizedInstall: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/containerized_installation
+:LinkContainerizedInstall: {URLContainerizedInstall}[{TitleContainerizedInstall}]
 //
 // titles/navigator-guide
 :TitleNavigatorGuide: Using content navigator
+:URLNavigatorGuide: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/using_content_navigator
+:LinkNavigatorGuide: {URLNavigatorGuide}[{TitleNavigatorGuide}]
 //
 // titles/aap-hardening
 :TitleHardening: Hardening and compliance
+:URLHardening: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/hardening_and_compliance
+:LinkHardening: {URLHardening}[{TitleHardening}]
 //
 // titles/builder
 :TitleBuilder: Creating and using execution environments
+:URLBuilder: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/creating_and_using_execution_environments
+:LinkBuilder: {URLBuilder}[{TitleBuilder}]
 //
 // titles/hub/managing-content
 :TitleHubManagingContent: Managing content in automation hub
+:URLHubManagingContent: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/managing_content_in_automation_hub
+:LinkHubManagingContent: {URLHubManagingContent}[{TitleHubManagingContent}]
 //
 // titles/analytics/job-explorer
 :TitleAnalyticsJobExplorer: Using automation analytics
+:URLAnalyticsJobExplorer: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/using_automation_analytics
+:LinkAnalyticsJobExplorer: {URLAnalyticsJobExplorer}[{TitleAnalyticsJobExplorer}]
 //
 // titles/develop-automation-content
 :TitleDevelopAutomationContent: Developing automation content
+:URLDevelopAutomationContent: {BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/developing_automation_content
+:LinkDevelopAutomationContent: {URLDevelopAutomationContent}[{TitleDevelopAutomationContent}]
 //
 // titles/topologies
 :TitleTopologies: Tested deployment models for Ansible Automation Platform
```

Comment:
I'm updating this today to "Tested deployment models". I can update this also depending on when it's merged 👍

Reply:
Rebased my PR against main to bring in this update.